CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Changelog for the Mapbox Maps SDK for Android
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
+## 9.4.0-beta.1 - August 6, 2020
```

Comment:
@pengdev are we still publishing releases with tag descriptions on https://github.com/mapbox/mapbox-gl-native-android/releases? We previously agreed that the GitHub release tag is where we'd list out what's in each pre-release, and in CHANGELOG.md is where we would combine all changes for one minor release (alpha, beta, stable) into a single entry

Reply:
Yes, we are still publishing releases with tag descriptions on https://github.com/mapbox/mapbox-gl-native-android/releases. 

I can refactor the CHANGELOG.md to combine all changes for one minor release in the follow-up PR.